internal/bridge/client.go

```diff
@@ -280,7 +280,7 @@ func (c *Client) doWithBackoff(
 			request.Header = headers.Clone()
 
 			//nolint:bodyclose // This response is returned to the caller.
-			response, err = c.Client.Do(request)
+			response, err = c.Do(request)
```

Comment:
:question: iiuc, this is ok because `Do` without `Client` effectively uses the default function?

Reply:
The linter is telling us to use using field promotion.